### PR TITLE
Reduce Azure CosmosDB RUs

### PR DESCRIPTION
**Pull request by gonzalogarciajaubert**
## Description
Reduce from 10.000 to 2.000 the default RU/s for CosmosDB

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- ~[ ] Updated documentation accordingly~

### CODE REVIEW

**Review by rdiaz82 — APPROVED** [review]
LGTM

File: packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-containers.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { toTerraformName } from '../helper/utils'
 import { BoosterConfig } from '@boostercloud/framework-types'
 import { eventsStoreAttributes } from '@boostercloud/framework-provider-azure'
 
+const MAX_CONTAINER_THROUGHPUT = 2000
```
Comment (NickSeagull) [inline packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-containers.ts:7]: Maybe we should move this into a file `constants.ts`?

**Review by NickSeagull — APPROVED** [review]
One small suggestion

File: packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-containers.ts (hunk 1 above)
Comment (gonzalogarciajaubert) [reply packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-containers.ts:7]: Ouch, merged before reading your comment @NickSeagull , sorry! I will move it in following PRs

**Pull request merged**